addons/website/static/src/components/translator/translator.js

```diff
@@ -254,7 +254,7 @@ export class WebsiteTranslator extends WebsiteEditorComponent {
 
         // Apply data-oe-readonly on nested data.
         $(this.websiteService.pageDocument).find(savableSelector)
-            .filter(':has(' + savableSelector + ')')
+            .filter(':has(' + savableSelector + ')' + ':not(div)')
```

Comment:
I don't know much about the previous code, did you blame further to see if there is any kind of explanation about why it is needed? "The other author that left Odoo" as you told me was just in charge of a big refactoring about this last year  but I doubt he was the one that introduce those lines initially.

In any case, I doubt this patch is the right one, especially without a huge comment above explaining why. The previous code apparently says "a savable area which contains a savable area should not be editable" (probably make sense to only allow editing the deepest savable areas... although I am not sure how/why this can happen and had to be protected like this)... changing that into "a savable area which contains a savable area should not be editable except if it is a div" would require comments to explain it.

Also, if this is indeed the right patch: why `')' + ':not(div)'` instead of `'):not(div)'`? But actually, this should probably be written like this (if this is the right solution):
```suggestion
            .has(savableSelector).not('div')
```